func.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/***************************************************************************
+ *                                                                          *
+ *   (c) 2004 Vladimir V. Kalynyak, Alexey V. Vinokurov, Ilya M. Shalnev    *
+ *                                                                          *
+ * This  is  commercial  software,  only  users  who have purchased a valid *
+ * license  and  accept  to the terms of the  License Agreement can install *
+ * and use this program.                                                    *
+ *                                                                          *
+ ****************************************************************************
+ * PLEASE READ THE FULL TEXT  OF THE SOFTWARE  LICENSE   AGREEMENT  IN  THE *
+ * "copyright.txt" FILE PROVIDED WITH THIS DISTRIBUTION PACKAGE.            *
+ ****************************************************************************/
+
+use Tygh\Enum\YesNo;
+
+defined('BOOTSTRAP') or die('Access denied');
+
+/**
+ * Displays information about the module on the module settings page
+ *
+ * @return string
+ */
+function fn_custom_carriers_info()
+{
+    $text = __('general_info');
```

Comment:
https://github.com/cscart/addon-carriers/pull/2/files#diff-5d3a31ef3994f3043e44fb8971fd0a6951967772ab0cfaeae868fc273ec98178R15

```suggestion
    $text = __('custom_carriers.general_info');
```